kitten/mouse picking/MousePicker.h

```diff
@@ -9,6 +9,7 @@ namespace MousePicker
 {
 	bool rayHits(const kitten::Ray& p_ray, const kitten::ClickableBox* p_clickable, double* p_hitOutput);	
 	bool uiHit(const kitten::ClickableFrame* p_clickable, float p_mouseX, float p_mouseY);
+	static bool m_isFocused = false;
```

Comment:
Could probably put this static bool in `DragNDrop` as it is only set and accessed there

Reply:
changed